server/channels/api4/ip_filtering.go

```diff
@@ -83,7 +83,18 @@ func applyIPFilters(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	auditRec.Success()
 
-	c.App.Srv().Go(func() {
+	cloudWorkspaceOwnerEmailAddress := ""
```

Comment:
nit: this "feels" like something that might belong in the app layer vs. directly in the api4 endpoint. Not blocking :)

Reply:
I've opened a ticket here: https://mattermost.atlassian.net/browse/CLD-6630 to address this in a future PR